integreat_cms/core/management/commands/repair_tree.py

```diff
@@ -0,0 +1,184 @@
+            self.print_error(
+                f"\ttree_id: {tree_node.tree_id} → {tree_node.parent.tree_id}"
+            )
+            valid = False
+        if tree_node.parent_id:
+            if tree_node.depth == tree_node.parent.depth + 1:
+                self.print_success(f"\tdepth: {tree_node.depth}")
+            else:
+                self.print_error(
+                    f"\tdepth: {tree_node.depth} → {tree_node.parent.depth + 1}"
+                )
+                valid = False
+        elif tree_node.depth == 1:
+            self.print_success(f"\tdepth: {tree_node.depth}")
+        else:
+            self.print_error(f"\tdepth: {tree_node.depth} → 1")
+            valid = False
+        if tree_node.lft == left:
+            self.print_success(f"\tlft: {tree_node.lft}")
+        else:
+            self.print_error(f"\tlft: {tree_node.lft} → {left}")
+            valid = False
+        if tree_node.rgt == right:
+            self.print_success(f"\trgt: {tree_node.rgt}")
+        else:
+            self.print_error(f"\trgt: {tree_node.rgt} → {right}")
+            valid = False
+        return valid
+
+    def check_for_orphans(self, tree_id):
+        """
+        Check whether orphans exist (pages with the same tree_id, but its ancestors are in another tree)
+
+        :param tree_id: The current tree id
+        :type tree_id: int
+        """
+        if orphans := Page.objects.filter(tree_id=tree_id).exclude(
+            id__in=self.pages_seen
+        ):
+            self.print_error(
+                "\n💣 Orphans detected! The following pages share the tree id "
+                f"{tree_id} but don't have a relation to the root node:"
+            )
+            for orphan in orphans:
+                self.stdout.write(self.bold(f"Page {orphan.id}:"))
+                self.print_error(
+                    f"\tparent_id: {orphan.parent_id}\n"
+                    f"\tparent.tree_id: {orphan.parent.tree_id}"
+                )
+                self.stdout.write(
+                    self.bold(
+                        f"\tdepth {orphan.depth}\n"
+                        f"\tlft: {orphan.lft}\n"
+                        f"\trgt: {orphan.rgt}"
+                    )
+                )
+
+    def calculate_left_right_values(self, tree_node, left, commit):
+        """
+        Recursively calculate the left and right value for a given node and its
+        children.
+
+        :param tree_node: A node of a MPTT tree
+        :type tree_node: ~integreat_cms.cms.models.pages.page.Page
+
+        :param left: The new left value of the node
+        :type left: int
+
+        :param commit: Whether changes should be written to the database
+        :type commit: bool
+
+        :return: The new right value of the node
+        :rtype: int
+        """
+        right = left
+
+        for child in tree_node.children.all():
+            right = self.calculate_left_right_values(child, right + 1, commit)
+
+        right += 1
+
```

Comment:
```suggestion

        valid = self.check_tree_fields(tree_node, left, right)

```
Sorry, somehow messed up my suggestions and forgot this call :sweat_smile:

Reply:
Just noticed. Thanks for providing the fix before I stumble over the problem ^^